lib/validators/properties-validator.ts

```diff
@@ -105,6 +106,17 @@ export class PropertiesValidator extends Validator {
         }
     }
 
+    private validateCheckBoxValue(property: ComponentProperty) {
```

Comment:
I wasn't really sure if this validation needs to be here or inside the schema. I guess we might be able to validate this with JSONSchema, but that might make the schema a bit complex/ugly.